src/aux_utils.cpp

```diff
@@ -640,6 +670,17 @@ namespace diskann {
     std::string medoids_path = disk_index_path + "_medoids.bin";
     std::string centroids_path = disk_index_path + "_centroids.bin";
     std::string sample_base_prefix = index_prefix_path + "_sample";
+    std::string disk_pq_pivots_path = index_prefix_path + "_disk.index_pq_pivots.bin"; // optional if disk index is also storing pq data
+    std::string disk_pq_compressed_vectors_path = // optional if disk index is also storing pq data
+        index_prefix_path + "_disk.index_pq_compressed.bin";
+
+// output a new base file which contains extra dimension with sqrt(1 - ||x||^2/M^2) for every x, M is max norm of all points. Extra space on disk needed!
+    if (compareMetric == diskann::Metric::INNER_PRODUCT) {
+      std::cout<<"Using Inner Product search, so need to pre-process base data into temp file. Please ensure there is sufficient space!!" << std::endl;
```

Comment:
CAn we be specific about extra space? Say create another file as large as base set?

Reply:
done. we need a note somewhere on total interim consumption also for all mem indices.